pages/event-resources.jsx

```diff
@@ -0,0 +1,407 @@
+var React = require('react');
+var ImageTag = require('../components/imagetag.jsx');
+
+var LinksContainer = React.createClass({
+  render: function() {
+    return (
+      <div className="row">{this.props.children}</div>
+    );
+  }
+});
+var PageLinker = React.createClass({
+  render: function() {
+    return (
+      <div className="page-linker col-sm-3 col-md-3 col-lg-3">
+        <a href={this.props.href}>{this.props.head}</a>
+        {this.props.children}
+      </div>
+    );
+  }
+});
+var EventItem = React.createClass({
+  render: function() {
+    return (
+      <div className="event-item">
+        <div className="event-icon">
+          <div className="participants-number">{this.props.participants}</div>
+          <div className="participants-label">PARTICIPANTS</div>
+        </div>
+        <div className="event-content">
+          <div className="event-item-header">{this.props.head}</div>
+          <div className="event-item-subheader">{this.props.subHead}</div>
+          <p className="event-item-content">{this.props.content}</p>
+        </div>
+      </div>
+    );
+  }
+});
+var LogoAsset = React.createClass({
+  render: function() {
+    return (
+      <div className="logo-asset-container col-sm-4 col-md-4 col-lg-3">
+        <div className="logo-asset-header">{this.props.head}</div>
+        <ImageTag className="logo-asset-img"
+        width={200} height={200}
+        src1x={this.props.src1x} src2x={this.props.src2x}
+        alt={this.props.alt}/>
+        <div className="logo-asset-hover">
+          <div className="logo-asset-center">{this.props.children}</div>
+        </div>
+      </div>
+    );
+  }
+});
+var LogoAssetLink = React.createClass({
+  render: function() {
+    return (
+      <a className="logo-asset-link" href={this.props.href}><span className="ion ion-ios-copy"></span>{this.props.children}</a>
+    );
+  }
+});
+var RemixLink = React.createClass({
+  render: function() {
+    return (
+      <a className="remix-link" href={this.props.href}>Remix</a>
+    );
+  }
+});
+var Tabulator = React.createClass({
+  getInitialState: function() {
+    return {
+      activeClass: "tab-0"
+    };
+  },
+  propTypes: {
+    tabs: React.PropTypes.array
+  },
+  showTab: function(activeTab) {
+    this.setState({
+      activeClass: activeTab
+    });
+  },
+  render: function() {
+    var tabulator = this;
+    var className = "tabulator";
+    if (this.state.activeClass) {
+      className += " " + this.state.activeClass;
+    }
+    return (
+      <div className={className}>
+        <div>
+          {this.props.tabs.map(function (section, key) {
+            function onClick(e) {
+              tabulator.showTab("tab-" + key);
+            }
+            var className = "tabulator-head col-sm-3 col-md-3 col-lg-3";
+            className += " tab-" + key;
+            return (
+              <div onClick={onClick} className={className} key={key}>{section.head}</div>
```

Comment:
Hmm ok. I kinda wonder if there is a way to just have screen-readers and such see the no-JS version of this tabset, so that it looks to them like a big ole page. That might actually be *more* accessible than making all the interactions work in an ARIA-compliant way, but I am no expert.

Anyhow sounds good!